### PR TITLE
Refactor:type safe feature option in dev tools

### PR DESCRIPTION
**Pull request by santoshyadav198613**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngrx/platform/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Documentation has been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[x] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

feature property was of type any.
Closes #

## What is the new behavior?
changed to interface to assign the type.
## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by santoshyadav198613** [comment]
Ready for review.

**Comment by santoshyadav198613** [comment]
> Thanks @santoshyadav198613, I am not sure if we want to type this as it is a passthrough to the Redux Dev Tools. If we moved forward, we would then need to stay in sync with the Redux Dev Tools options object.
> 
> Open to debate it though...
> 
> Here are the current features in Redux Dev Tools:
> 
> https://github.com/zalmoxisus/redux-devtools-extension/blob/master/docs/API/Arguments.md#features

Yes referred the same, I think they are not changing the features, even the development is hardly happening at there end. After July they hardly have any release.

**Comment by brandonroberts** [comment]
We don't want to create an explicit dependency on the Redux DevTools, so the interface here is fine.

File: modules/store-devtools/src/config.ts [hunk 1]
```diff
@@ -11,6 +11,18 @@ export type SerializationOptions = {
   refs?: Array<any>;
 };
 export type Predicate = (state: any, action: Action) => boolean;
+export interface IDevToolsFeature {
```
[the remark this answers is not in the document]
Comment (santoshyadav198613) [reply modules/store-devtools/src/config.ts]: Done.

**Pull request merged**